ac/ac-imgClass/src/imgClassifier.jsx

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import React from 'react';
+import Mousetrap from 'mousetrap';
+import type { ActivityRunnerT } from 'frog-utils';
+import styled from 'styled-components';
+
+const ImgBis = styled.img`
+  max-width: 80%;
+  max-height: 100%;
+  position: relative;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+const FlexDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 90%;
+`;
+
+const ImgPanel = (props: { url: string }) =>
+  <div style={{ width: '90%', height: '100%' }}>
+    <ImgBis alt="" src={props.url} />{' '}
+  </div>;
+
+const ShortCutPanel = (props: { categories: Array<string> }) =>
+  <ul className="list-group" style={{ width: '20%' }}>
+    {props.categories.map((x, i) =>
+      <li key={x} className="list-group-item">
+        {' '}{i + 1} <span className="glyphicon glyphicon-arrow-right" /> {x}{' '}
+      </li>
+    )}
+  </ul>;
+
+export default ({ activityData, data, dataFn }: ActivityRunnerT) => {
+  const categ = activityData.config.categories || [];
+  const imgs = activityData.config.images || [];
+  categ.forEach((x, i) =>
+    Mousetrap.bind((i + 1).toString(), () => {
+      dataFn.objInsert(x, data.index + 1);
```

Comment:
Right now the reactive data looks like this: 
```JavaScript
{
  1: "Light Side",
  2: "Light Side",
  3: "Light Side",
  4: "Light Side",
  5: "Light Side",
  6: "Light Side",
  7: "Light Side",
  index: 7
}
```
And the corresponding image urls are in the config of the activity. In our dataflow, the config of the activity is not transfered to the product, so operators that would use the output of this activity, would not know the urls. I believe the following format will be more usable:

```JavaScript
{
  1: { url: "...", category: "Light Side"},
  ...,
  index: 7
}
```

Stian mentionned in #388:
>  The output of the activity is the same list of URLs, with the tag received by the user.